### PR TITLE
#184 #195 Add flow-resources with StringDesc bindings for Flow

### PR DESCRIPTION
**Pull request by kovalandrew**

### CODE REVIEW

File: mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import android.widget.TextView
+import androidx.lifecycle.LifecycleOwner
+import dev.icerock.moko.mvvm.flow.CStateFlow
+import dev.icerock.moko.mvvm.flow.binding.bind
+import dev.icerock.moko.resources.desc.StringDesc
+import kotlinx.coroutines.DisposableHandle
+
+fun TextView.bindText(
+    lifecycleOwner: LifecycleOwner,
+    flow: CStateFlow<StringDesc>
```
Comment (Alex009) [inline mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt:16]: we should also support nullable StringDesc

Comment (Alex009) [reply mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt:16]: just like in `UIButton` extension

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt (hunk 1 above)
Comment (kovalandrew) [reply mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt]: @Alex009 Yeah, sure. Fixed

File: mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import androidx.lifecycle.LifecycleOwner
+import dev.icerock.moko.mvvm.flow.CStateFlow
+import dev.icerock.moko.mvvm.flow.binding.bind
+import dev.icerock.moko.resources.desc.StringDesc
+import kotlinx.coroutines.DisposableHandle
+
+fun TextView.bindText(
+    lifecycleOwner: LifecycleOwner,
+    flow: CStateFlow<StringDesc?>
+): DisposableHandle {
```
Comment (Alex009) [inline mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt:17]:
```
fun <T : StringDesc?> TextView.bindText(
     lifecycleOwner: LifecycleOwner,
     flow: CStateFlow<T>
 ): DisposableHandle
```

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt (hunk 2 above)
Comment (kovalandrew) [reply mvvm-flow-resources/src/androidMain/kotlin/dev/icerock/moko/mvvm/flow/resources/TextViewBindings.kt:17]: Fixed

**Review by Alex009 — APPROVED** [review]

**Pull request merged**